srcpkgs/dracut/template

```diff
@@ -52,8 +67,19 @@ dracut-network_package() {
 	archs=noarch
 	pkg_install() {
 		for f in 40network 90livenet 95fcoe 95iscsi \
-		    95nbd 95nfs 95zfcp 95znet; do
+			95nbd 95nfs 95zfcp 95znet; do
 			vmove usr/lib/dracut/modules.d/${f}
 		done
 	}
 }
+
+dracut-uefi_package() {
+	depends="binutils gummiboot dracut-${version}_${revision}"
```

Comment:
Why does it depend on `gummibot`? It’s not needed for booting Linux via EFISTUB.

Reply:
The dracut UEFI bundle requires a separate UEFI stub provided (in our case) by gummiboot.